velox/docs/functions/spark/math.rst

```diff
@@ -6,6 +6,9 @@ Mathematical Functions
 
     Returns the absolute value of ``x``.
 
+.. spark:function:: acos(x) -> double
+    Returns the inverse cosine (a.k.a. arc cosine) of ``x``.
```

Comment:
Would you check the generated docs? I feel you may need to add an empty line before this.

Reply:
done